README.md

```diff
@@ -1,52 +1,104 @@
 
-DBOS Transact is a **Python library** for building durable and scalable applications.
+DBOS Transact is a **Python library** providing ultra-lightweight durable execution.
+For example:
+
+```python
+@DBOS.step()
+def step_one():
+    ...
+
+@DBOS.step()
+def step_two():
+    ...
+
+@DBOS.workflow()
+def workflow()
+    step_one()
+    step_two()
+```
+
+Durable execution means your program is **resilient to any failure**.
+If it is ever interrupted or crashes, all your workflows will automatically resume from the last completed step.
+If you want to see durable execution in action, check out [this demo app](https://demo-widget-store.cloud.dbos.dev/) (source code [here](https://github.com/dbos-inc/dbos-demo-apps/tree/main/python/widget-store)).
+No matter how many times you try to crash it, it always resumes from exactly where it left off!
+
+Under the hood, DBOS Transact works by storing your program's execution state (which workflows are currently executing and which steps they've completed) in a Postgres database.
+So all you need to use it is a Postgres database to connect to&mdash;there's no need for a "workflow server."
+This approach is also incredibly fast, for example [25x faster than AWS Step Functions](https://www.dbos.dev/blog/dbos-vs-aws-step-functions-benchmark).
 
-You want to use DBOS Transact in your application because you need:
+Some more cool features include:
 
-- **Resilience to any failure**.  If your app is interrupted for any reason, it automatically resumes from where it left off.  Reliable message delivery is built in. Idempotency is built in.
-- **Reliable event processing**. Need to consume Kafka events exactly-once? Just add one line of code to your app. Need to run a task exactly once per hour, day, or month? Just one more line of code.
-- **Built-in observability**. Automatically emit [OpenTelemetry](https://opentelemetry.io/)-compatible logs and traces from any application. Query your app's history from the command line or with SQL.
-- **Blazing-fast, developer-friendly serverless**.  Develop your project locally and run it anywhere. When you're ready, [deploy it for free to DBOS Cloud](https://docs.dbos.dev/getting-started/quickstart#deploying-to-dbos-cloud) and we'll host it for you, [25x faster](https://www.dbos.dev/blog/dbos-vs-aws-step-functions-benchmark) and [15x cheaper](https://www.dbos.dev/blog/dbos-vs-lambda-cost) than AWS Lambda.
+- Scheduled jobs&mdash;run your workflows exactly-once per time interval.
+- Exactly-once event processing&mdash;use workflows to process incoming events (for example, from a Kafka topic) exactly-once.
+- Observability&mdash;all workflows automatically emit [OpenTelemetry](https://opentelemetry.io/) traces.
 
 ## Getting Started
 
-To try out the latest pre-release version, install with:
+To try out the latest pre-release version, install and configure with:
 
 ```shell
 pip install --pre dbos
+dbos init --lightweight
```

Comment:
I'd suggest `--config` because the lightweight mode only copies the config file.